src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        createFileWithGroupedData(toFileName, createGroupedData(extractDataFromFile(fromFileName)));
+    }
+
+    private static String extractDataFromFile(String fileName) {
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            String dataFromFile = reader.readLine();
+            while (dataFromFile != null) {
+                builder.append(dataFromFile).append(" ");
+                dataFromFile = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can not read file " + fileName);
+        }
+        return builder.toString();
+    }
+
+    private static String[] createGroupedData(String dataInLine) {
+        String[] values = dataInLine.split(" ");
+        int sumOfSupply = 0;
+        int sumOfBuy = 0;
+        for (String value : values) {
+            String[] splitValue = value.split(",");
```

Comment:
let's declare "," as CSV_SEPARATOR, that will improve readability
```suggestion
            String[] splitValue = value.split(CSV_SEPARATOR);
```